site/app/controllers/admin/UsersController.php

```diff
@@ -636,7 +636,12 @@ public function updateRotatingSections() {
             }
             $curr_section_sizes = $this->core->getQueries()->getUsersCountByRotatingSections();
             $section_assignment_counts = array_map(function ($expected_size, $curr_size) {
-                return $expected_size - $curr_size['count'];
+                if ($curr_size === null) {
+                    return $expected_size;
+                }
+                else {
+                    return $expected_size - $curr_size['count'];
+                }
```

Comment:
This could be replaced with a ternary expression.
```suggestion
                return $curr_size === null ? $expected_size : $expected_size - $curr_size['count'];
```